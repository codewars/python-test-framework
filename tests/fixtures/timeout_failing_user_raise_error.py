import codewars_test as test


@test.describe("group 1")
def group_1():
    @test.it("it 1")
    def group_1():
        @test.timeout(0.2)
        def test_1():
            raise KeyError()
            test.pass_()
