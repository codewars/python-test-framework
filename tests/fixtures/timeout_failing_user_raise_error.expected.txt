
<DESCRIBE::>group 1

<IT::>it 1

<FAILED::>Should not throw any exceptions inside timeout: KeyError()

<COMPLETEDIN::>30.41
